Tests/BugsnagTests/ReporterTests.swift

```diff
@@ -321,6 +322,17 @@ class ReporterTests: XCTestCase {
         XCTAssertEqual(config.stackTraceSize, 100)
     }
 
+    func testThatNotifyReleaseStagesAcceptsNilValueInConfig() {
```

Comment:
I believe this test belongs in `ConfigurationTests` (not sure if we created that already), since this is not about reporting, but about setting up the configuration.